sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -159,7 +160,9 @@ SUBROUTINE CLDRAD
       real                     :: ceiling_thresh_cldfra, cldfra_max, &
                                   zceil, zceil1, zceil2, previous_sum, &
                                   ceil_min, ceil_neighbor
-      real,dimension(im,jm)    :: ceil
+
+      real,dimension(im,jsta:jend)    :: ceil
```

Comment:
@BoCui-NOAA This array is also used by the GSD specified computation 711 and 798. I would suggest to keep the full  domain declaration at this time.

Reply:
I will take a look and reverse the change. Thanks!